svgdigitizer/electrochemistry/cv.py

```diff
@@ -312,6 +317,114 @@ def x_label(self):
             match[0], match[1], match[2] or "unknown"
         )
 
+    @property
+    @cache
+    def figure_label(self):
+        r"""
+        Return the figure name.
+
+        The figure name is read from a ``<text>`` in the SVG file
+        such as ``<text>figure: 2b</text>``.
+
+        Besides `figure`, also `fig` is acceptable in the text field.
+
+        EXAMPLES::
+
+            >>> from svgdigitizer.svg import SVG
+            >>> from svgdigitizer.svgplot import SVGPlot
+            >>> from svgdigitizer.electrochemistry.cv import CV
+            >>> from io import StringIO
+            >>> svg = SVG(StringIO(r'''
+            ... <svg>
+            ...   <g>
+            ...     <path d="M 0 200 L 0 100" />
+            ...     <text x="0" y="200">x1: 0 cm</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M 100 200 L 100 100" />
+            ...     <text x="100" y="200">x2: 1cm</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M -100 100 L 0 100" />
+            ...     <text x="-100" y="100">y1: 0</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M -100 0 L 0 0" />
+            ...     <text x="-100" y="0">y2: 1 A</text>
+            ...   </g>
+            ...   <text x="-200" y="330">Figure: 2b</text>
+            ... </svg>'''))
+            >>> cv = CV(SVGPlot(svg))
+            >>> cv.figure_label
+            '2b'
+
+        """
+        figure_labels = self.svgplot.svg.get_texts("(?:figure|fig): (?P<label>.+)")
+
+        if not figure_labels:
+            return ""
+
+        if len(figure_labels) > 1:
+            logger.warning(
+                f"More than one text field with figure labels. Ignoring all text fields except for the first: {figure_labels[0]}."
+            )
+
+        return figure_labels[0].label
+
+    @property
+    @cache
+    def curve_label(self):
+        r"""
+        Return the curve label.
```

Comment:
```suggestion
        A descriptive label for this curve to distinguish it from other curves in the same plot.
```

A property should not say "return" imho. And we should try not just to repeat what the method name already says.

Reply:
Makes sense. I did not think about that.